lib/ifc/analysis-utils.ts

```
@@ -1,89 +1,155 @@
-        details: Array(clashCount)
-          .fill(0)
-          .map((_, i) => ({
-            id: `clash-${i}`,
-            element1: `element-${Math.floor(Math.random() * elements.length)}`,
-            element2: `reference-${Math.floor(Math.random() * referenceElements.length)}`,
-            distance: Math.random() * tolerance,
-          })),
-      }
+    case "space":
+      // Space analysis using IfcOpenShell
+      const metric = options.metric || "room_assignment"
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)